src/main/kotlin/no/nav/syfo/model/DialogmeldingMapper.kt

```diff
@@ -71,7 +71,7 @@ fun XMLNotat.toHenvendelseFraLegeHenvendelse(): HenvendelseFraLegeHenvendelse {
 }
 
 fun CV.toTeamakode(): TemaKode {
-    return findDialogmeldingKodeverk(s, v).toTypeMelding()
+    return findDialogmeldingKodeverk(s, v)!!.toTypeMelding()
```

Comment:
Hm, vil ikke !! kunne gå i nullpointer dersom man ikke finner kodeverk?

Ellers, hva betyr variablene "s" og "v"?

Reply:
Hvis kodeverk blir null, vil man ikke komme hit, så det skal gå bra 👍 Navnet på felter og sånt er litt rart, det kommer fra xml'en.  "s" er IDen på kodeverket, og "v" tror jeg står for verdi. IDen gir deg et kodeverk, og så kan man ha flere verdier.
https://volven.no/produkt.asp?id=445915&catID=3&subID=8